python_modules/dagster/dagster_tests/general_tests/compat_tests/test_back_compat.py

```diff
@@ -1225,6 +1226,10 @@ def test_add_backfill_id_column():
             )
 
             instance.upgrade()
+            # sqlite storage .from_local runs the migrations on instantiation. This means the data migration
+            # doesn't get run when we call instance.upgrade. So we manually run the migration here to test that
+            # it works
```

Comment:
maybe add a TODO here to figure out why it's getting stamped?  I'd think that the local sqlite file would also have the alembic version in there.